zerver/lib/webhooks/git.py

```diff
@@ -273,6 +276,32 @@ def get_issue_event_message(
     )
 
 
+def get_issue_labeled_or_unlabeled_event_message(
```

Comment:
Yes, I also think that this kind of refactoring makes sense since most of the functions defined in this file are not explicitly related to Git. We can create a follow-up issue for that.